packages/chain-adapters/package.json

```diff
@@ -25,6 +25,7 @@
     "type-check": "tsc --project ./tsconfig.json --noEmit"
   },
   "dependencies": {
+    "@shapeshiftoss/caip": "^1.5.0",
```

Comment:
I think you'll want to put this in the peerDependencies object. Helps with versioning.

Reply:
Yep, good call. I see those changes now, thanks!

Reply:
Is there a reason for things to be both devDependecies and peerDependencies? or is that leftover cruft?

Reply:
That's a good question, I think it needs to be in both. @majorhayes added it to both when he made these changes last week.

Reply:
A little googling and it seems like peer deps are installed by default. I tried the --peer flag with yarn and that also didn't seem to work. Seems like there is a package that can do this for us, but not a huge deal right now so just adding the dupes